tensorflow/lite/micro/tools/ci_build/test_size.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Build a TFLite micro test binary and capture its size.
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_DIR=${SCRIPT_DIR}/../../../../..
+cd "${ROOT_DIR}"
+
+
+SIZEFILE_DIR=${ROOT_DIR}/ci
+MAKEFILE_DIR=${ROOT_DIR}/tensorflow/lite/micro/tools/make
+BENCHMARK_TARGET=binary_size_test
```

Comment:
Nit: this isn't really a benchmark right? s/BENCHMARK/TEST?

Reply:
You are right that this right now just points to a test binary. 
The intention is that this will become the benchmark target when we have a defined benchmark target for size.  

I will add more comments in PR description.

Reply:
Ack.